src/context/WalletProvider/NativeWallet/components/NativePassword.tsx

```diff
@@ -34,10 +42,10 @@ export const NativePassword = ({ history }: RouteComponentProps) => {
   return (
     <>
       <ModalHeader>
-        <Text translation={'walletProvider.shapeShift.nativePasswd.header'} />
+        <Text translation={'walletProvider.shapeShift.password.header'} />
       </ModalHeader>
       <ModalBody>
-        <Text mb={6} color='gray.500' translation={'walletProvider.shapeShift.nativePasswd.body'} />
+        <Text mb={6} color='gray.500' translation={'walletProvider.shapeShift.password.body'} />
         <form onSubmit={handleSubmit(onSubmit)}>
```

Comment:
It would probably be a decent idea to make this component revocable as well... would suck if someone could pull the *password itself* out of react state.

Reply:
I don't think there's such a think as "state history". Once the component is unmounted, the state is gone, not just hidden.